tests/saitests/sai_qos_tests.py

```diff
@@ -672,6 +673,9 @@ def runTest(self):
         # get counter names to query
         ingress_counters, egress_counters = get_counter_names(sonic_version)
 
+        # Clear all counters for all ports
+        sai_thrift_clear_all_counters(self.client)
```

Comment:
Not sure if this works for all vendors. Can you take the counter snapshot prior to the test run and after and compare as is done for all the testcases?

Reply:
Done